mobify/platform-scaffold
# Fix checkout confirmation reducer

# Fix checkout confirmation reducer

The reducer for checkout confirmation was incorrectly looking at the thunk action instead of `realReceiveCheckoutConfirmationData`

 **JIRA**: N/A
 **Linked PRs**: https://github.com/mobify/platform-scaffold/pull/556

## Changes
- Exported correct confirmation action
- Added above action to checkout confirmation reducer

<img width="317" alt="screen shot 2017-06-01 at 11 16 13 am" src="https://cloud.githubusercontent.com/assets/16109837/26694282/c12099a8-46bb-11e7-8cc3-6c52b4ff1614.png">

## How to test-drive this PR
- SFCC Connector
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Ensure a full checkout flow results in a confirmation page
